pkg/rules/rules.go

```diff
@@ -248,3 +250,26 @@ func (srv *rulesServer) Send(res *rulespb.RulesResponse) error {
 func (srv *rulesServer) Context() context.Context {
 	return srv.ctx
 }
+
+func CheckRulesFiles(filePaths []string) error {
```

Comment:
I dont see that this is called anywhere; why is it needed?

Reply:
just like the issue I mentioned above, we need a method to replace `rulefmt.ParseFile` in prometheus since thanos change the alert rule file and and `rulefmt.ParseFile` cannot chech the changed file.

Reply:
Is there any way we could use it in the code-base itself? I'm afraid that this logic will get stale.

Reply:
no, the key func is `ValidateAndCount()`, which is used by thanos CLI tool,too.